internal/uvm/create_wcow.go

```diff
@@ -292,7 +292,11 @@ func CreateWCOW(ctx context.Context, opts *OptionsWCOW) (_ *UtilityVM, err error
 			},
 		}
 
-		uvm.scsiLocations[0][0] = newSCSIMount(uvm, doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path, "", "", 1, 0, 0, false)
+		uvm.scsiLocations[0][0] = newSCSIMount(uvm,
+			doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path,
+			"",
+			doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Type_,
+			"", 1, 0, 0, false)
```

Comment:
I'm not actually sure what the idiomatic approach for formatting function calls that span multiple lines is (if one line was getting too long, e.g. this one). 

To me this seems strange to have some on new lines, some not, and then a final stretch at the end with 5 values on one line. 

How about this (unless there's a recommended style approach for this situation somewhere):

```go 
uvm.scsiLocations[0][0] = newSCSIMount(
                  uvm,
	          doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path,
		  "",
		  doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Type_,
		  "", 
                  1, 
                  0, 
                  0, 
                  false,
)

Reply:
Yeah, I wasn't sure about this either and I didn't find any recommended style anywhere else. So I have changed it to one argument per line.